tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1540,11 +1536,7 @@ def _rostopic_cmd_bw(argv=sys.argv):
     if len(args) > 1:
         parser.error("you may only specify one input topic")
     try:
-        if options.window_size:
-            import string
-            window_size = string.atoi(options.window_size)
-        else:
-            window_size = options.window_size
+        window_size = int(options.window_size) if options.window_size else None
```

Comment:
If that is what the user asked for why do something else? This magic would then also need to be documented.